src/controllers/commands/CommandController.cpp

```diff
@@ -488,6 +488,8 @@ CommandController::CommandController(const Paths &paths)
     this->registerCommand("/endpoll", &commands::endPoll);
 
     this->registerCommand("/prediction", &commands::createPrediction);
+    this->registerCommand("/cancelprediction", &commands::cancelPrediction);
+    this->registerCommand("/endprediction", &commands::endPrediction);
```

Comment:
this or `/lockprediction`?

Reply:
`/lockprediction` 100%. I feel like the naming on Twitch's docs is confusing. "End Prediction" should be something like "Update Prediction". We should also mention in our code that `endPrediction` doesn't really end a prediction, but that it locks, resolves, or cancels one.

Reply:
"End Submissions" is the phrasing Twitch uses, so I'd be fine with "/endpredictionsubmissions" too